Nbuild/resources/common.targets

```diff
@@ -122,6 +123,7 @@
 		<Message Text="==> DONE"/>
 	</Target>
 
+	<!-- Standardize on dev, stage, and prod environments and phase out STAGING and PRODUCTION -->
```

Comment:
This is a good comment for a TODO. Not sure if you want it merged. Looks good otherwise.

Reply:
Thanks @ahmdhjj  I'm going to make this refactoring in the branch and update the documents